src/lib/mina_base/zkapp_command.ml

```diff
@@ -415,10 +415,11 @@ let check_authorization (p : Account_update.t) : unit Or_error.t =
 
 module Verifiable : sig
   type t =
-    (Side_loaded_verification_key.t, Zkapp_basic.F.t) With_hash.t option
+    ( Proof.t
```

Comment:
Note: this is originally `Pickles.Side_loaded.Proof.Stable.V2.t = Verification_key.Max_width.n, Verification_key.Max_width.n) Proof.t
`, now replaced with`Pickles.Proof.Proofs_verified_2.Stable.V2.t`. Samething under the hood.